docs/csharp/whats-new/csharp-6.md

```diff
@@ -98,483 +36,157 @@ public class Student
-that tools can understand the symbol. If you use refactoring tools to
-rename the symbol, it will rename it in the `nameof` expression. Constant
-strings don't have that advantage. Try it yourself in your favorite editor:
-rename a variable, and any `nameof` expressions will update as well.
-
-The `nameof` expression produces the unqualified name of its argument
-(`LastName` in the previous examples) even if you use the fully qualified
-name for the argument:
-
-[!code-csharp[QualifiedNameofNotify](../../../samples/snippets/csharp/new-in-6/viewmodel.cs#QualifiedNameofNotify)]
-
-This `nameof` expression produces `FirstName`, not `UXComponents.ViewModel.FirstName`.
-
 ## Await in Catch and Finally blocks
 
-C# 5 had several limitations around where you could place `await` expressions.
-One of those has been removed in C# 6. You can now use `await` in `catch`
-or `finally` expressions. 
-
-The addition of await expressions in catch and finally blocks may appear
-to complicate how those are processed. Let's add an example to discuss
-how this appears. In any async method, you can use an await expression
-in a finally clause.
-
-With C# 6, you can also await in catch expressions. This is most often
-used with logging scenarios:
+C# 5 had several limitations around where you could place `await` expressions. You can now use `await` in `catch` or `finally` expressions.  With C# 6, you can also await in catch expressions. This is most often used with logging scenarios:
 
 [!code-csharp[AwaitFinally](../../../samples/snippets/csharp/new-in-6/NetworkClient.cs#AwaitFinally)]
 
-The implementation details for adding `await` support inside `catch`
-and `finally` clauses ensures that the behavior is consistent with the
-behavior for synchronous code. When code executed in a `catch` or `finally`
-clause throws, execution looks for a suitable `catch` clause in the next
-surrounding block. If there was a current exception, that exception is
-lost. The same happens with awaited expressions in `catch` and `finally`
-clauses: a suitable `catch` is searched for, and the current exception,
-if any, is lost.  
+The implementation details for adding `await` support inside `catch` and `finally` clauses ensure that the behavior is consistent with the behavior for synchronous code. When code executed in a `catch` or `finally` clause throws, execution looks for a suitable `catch` clause in the next surrounding block. If there was a current exception, that exception is lost. The same happens with awaited expressions in `catch` and `finally` clauses: a suitable `catch` is searched for, and the current exception, if any, is lost.  
 
 > [!NOTE]
-> This behavior is the reason it's recommended to write `catch` and `finally`
-> clauses carefully, to avoid introducing new exceptions.
+> This behavior is the reason it's recommended to write `catch` and `finally` clauses carefully, to avoid introducing new exceptions.
 
 ## Index initializers
```

Comment:
I think the problem is is just with the refactor of info. This section is **Index Initializers** but you talk about **Collection Initializers** by referring to braces and the `Add` method. Also, the first code sample shows using a `List` type which isn't using index initializers.

Reply:
OK just to clarify, this is pointing to the `## Index initializers` line and below. Not the change above that :D